lib/presentation/feature/splash/splash_screen.dart

```diff
@@ -19,7 +19,7 @@ class _SplashScreenState extends State<SplashScreen> {
   @override
   void initState() {
     _timer = Timer(const Duration(seconds: 2), () {
-      context.router.navigate(HomeRoute());
+      context.router.replace(const MainRoute());
```

Comment:
use from build context when possible
```dart
context.pushRoute(...)
context.popRoute(...)
```

Reply:
bhai, not getting the point, would you please elaborate a bit please